ibis/pandas/execution/window.py

```diff
@@ -90,6 +90,10 @@ def execute_window_op(
         **kwargs,
     )
 
+    if window.max_lookback is not None:
+        raise NotImplementedError('Rows with max lookback is not implemented '
+                                  'for pandas backends.')
```

Comment:
I think you can use singular `backend` here

Reply:
Fixed in next commit